discovery/gossiper.go

```diff
@@ -514,12 +514,52 @@ func (d *AuthenticatedGossiper) start() error {
 
 	d.syncMgr.Start()
 
+	// Start receiving blocks in its dedicated goroutine.
+	d.wg.Add(1)
```

Comment:
nit: can put this below and `d.wg.Add(2)`?

Reply:
Switched.